### PR TITLE
add missing OC attributes

### LINKED ISSUE #7705: Implement new OC attributes
This is a follow up to #7671 
There we had some OC attributes not mapped to GC attributes.

Furthermore, some OC attribute definitions of 
https://wiki.opencaching.eu/index.php?title=Cache_attributes
are not implemented in c:geo.
At the moment these are the following OCPL only attributes:

ID | text_short | text_long | id | gc_id | icon | name | gc_inc | ACODE | OCGS
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
214 | Handicaching | Rated on Handicaching.com |   |   |   |   |   | A76 | 214
215 | Munzee | Contains a Munzee |   |   |   |   |   | A77 | 215
216 | Ads | Contains advertising |   |   |   |   |   | A78 | 216
217 | Military training area | Military training area, some access restrictions - check before visit |   |   |   |   |   | A79 | 217
218 | Video surveillance | Caution, area under video surveillance |   |   |   |   |   | A80 | 218
219 | Trackables | Suitable to hold trackables |   |   |   |   |   | A81 | 219
54 | Ruin | Abandoned structure / ruin |   |   |   |   |   | A82 |
48 | UV light | UV light required |   |   |   |   |   | A83 |
9015 | NOT in winter | NOT available during winter |   |   |   |   |   | A84 |  
9001 | No dogs | Dogs not allowed |   |   |   |   |   | A85 |  
46 | Truck / RV | Truck / RV allowed |   |   |   |   |   | A86 |
220 | Historic | Officially designated historical monument |   |   |   |   |   | A87 | 220

### PR DESCRIPTION
**Pull request by bekuno**
fixes #7705

In the part for `attributes_okapi` I decided to remove the old eclipse configuration
and switch to manual compilation for the attribute_definition.xml analyze tool.

This tool is updated for all OC location and results in more information.

The readme.txt covers the changes.

The attribute_definition.xml was updated from the current okapi version.

In the part `attributes` the icon workflow was refreshed.
This is covert in the readme.txt there.

The shell scripts got some code cleanup.
Only the cache icon overview script changes to present the acode instead of the OCDE code.

The OCDE icons overview file was refreshed and renamed to make clear the OCDE reference.
It got two additional attribute lines.

For the 8 new OC attributes I created the attribute icons based on PD/CC licenced vector images.

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1657,6 +1657,22 @@
...
+    <string name="attribute_munzee_yes">Contains a Munzee</string>
+    <string name="attribute_munzee_no">Contains no Munzee</string>
+    <string name="attribute_ads_yes">Contains advertising</string>
+    <string name="attribute_ads_no">Contains no advertising</string>
+    <string name="attribute_military_area_yes">Military training area</string>
+    <string name="attribute_military_area_no">No military training area</string>
+    <string name="attribute_video_surveil_yes">Video surveillance</string>
+    <string name="attribute_video_surveil_no">No video surveillance</string>
+    <string name="attribute_trackables_yes">Trackables</string>
+    <string name="attribute_trackables_no">Trackables</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:1671]: Missing "No"?

Comment (bekuno) [reply main/res/values/strings.xml]: > Missing "No"?

Yes.

File: main/project/attributes/readme.txt [hunk 2]
```diff
@@ -45,8 +45,9 @@ Icons are written to directory ./drawable-mdpi. Copy them to res/drawable-mdpi w
 
 Add a new line or change an existing one.
 Don't mess up the structure of this file. It is used to automatically create missing strings, html-pages and enumerations.
-The first column contains the name of the attribute. In case of gc.com, this also is their internal id and is used for parsing websites in c:geo. So don't change any IDs!
+The first column contains the name of the attribute. In case of geocaching.com, this also is their internal id and is used for parsing websites in c:geo. So don't change any IDs!
 The second is the "groundspeak:attribute id" from a gpx file. You can get them from a cache gpx export.
+The third is the ACOCDE, the "opencaching id". This you will get from their attribute-definitions.xml. To prepare this part follow ../attributes_okapi/readme.txt.
```
Comment (rsudev) [inline main/project/attributes/readme.txt:50]: Minimal typo: ACODE
```suggestion
The third is the ACODE, the "opencaching id". This you will get from their attribute-definitions.xml. To prepare this part follow ../attributes_okapi/readme.txt.
```

**Review by rsudev — APPROVED** [review]

File: main/project/attributes/readme.txt (hunk 2 above)
Comment (bekuno) [reply main/project/attributes/readme.txt]: Fixed.

**Pull request merged**